crates/matrix-sdk-base/src/store/send_queue.rs

```diff
@@ -131,6 +132,9 @@ pub struct QueuedRequest {
     /// The bigger the value, the higher the priority at which this request
     /// should be handled.
     pub priority: usize,
+
+    /// The time that the request was original attempted.
+    pub enqueue_time: Option<MilliSecondsSinceUnixEpoch>,
```

Comment:
What do you think of calling this (everywhere, so in the SQL too) `creation_time`, or even `created_at`?

Reply:
Love that name, will add